build.gradle

```diff
@@ -47,16 +47,16 @@ ext {
                                   },
             slf4j               : [group: 'org.slf4j', name: 'slf4j-api', version: '1.7.30'],
             // log4j 2.13.0 requires Java 8.
```

Comment:
If we do upgrade to `2.17.0`, we don't need this comment (which was based on https://logging.apache.org/log4j/2.x/index.html#Requirements) anymore.

Reply:
Yes, but seeing this comment I expected to run into some trouble that I wasn't sure I can fix :wink: (and I'm still not sure -> CI)